common/tools/dev-tool/src/util/resolveProject.ts

```diff
@@ -136,7 +137,7 @@ async function findAzSDKPackageJson(directory: string): Promise<[string, Package
 
   for (const file of files) {
     if (file === "package.json") {
-      const fullPath = path.join(directory, file);
+      const fullPath = pathToFileURL(path.join(directory, file)).href;
```

Comment:
On windows: Error: Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'd:'

This is not necessary for linux / mac